arch/xtensa/Kconfig

```diff
@@ -173,4 +173,11 @@ endif # XTENSA_MMU
 
 endif # CPU_HAS_MMU
 
+config ARCH_CPU_IDLE_CUSTOM
```

Comment:
this should not be xtensa specific, please move it up so any other architecture can do the same if needed.

Reply:
@nashif  updated, thanks